sdk/core/core/test/cmocka/CMakeLists.txt

```diff
@@ -19,24 +19,15 @@ endif()
 
 add_cmocka_test(${TARGET_NAME} SOURCES
                 main.c
-                test_az_json.c
-                test_json_value.c
-                test_json_string.c
-                test_json_pointer.c
-                test_json_parser.c
-                test_json_get_by_pointer.c
-                test_json_builder.c
-                test_az_span.c
-                test_span.c
-                test_span_replace.c
+                test_az_aad.c
```

Comment:
We should add a comment here to let future contributors know to maintain this list in alphabetical order.